datastore/utils/src/test/groovy/org/orbisgis/datastore/utils/JDBCDataStoreUtilsTest.groovy

```diff
@@ -48,14 +50,61 @@ import org.junit.jupiter.api.Test
  */
 class JDBCDataStoreUtilsTest {
 
+    private static JDBCDataStore ds
+
+    @BeforeAll
+    static void beforeAll() {
+        def dataStore = DataStoreFinder.getDataStore([dbtype: "h2gis", database: "./target/database_${UUID.randomUUID()}"])
+        assert dataStore in JDBCDataStore
+        ds = (JDBCDataStore) dataStore
+        ds.dataSource.getConnection().createStatement().execute("""
+            CREATE TABLE elements (
+                id int,
+                name varchar(255),
+                number int
+            );
+            INSERT INTO elements (id, name, number) VALUES (1, 'Simple Name', 2846);
+            INSERT INTO elements (id, name, number) VALUES (2, 'Maybe a complex Name', 7455);
+            INSERT INTO elements (id, name, number) VALUES (3, 'S N', 9272);
+        """)
+    }
+
     @Test
-    void missingPropertyTest() {
-        def name = "landcover2000.shp"
-        assert this.class.getResource(name)
-        def shapefile = new ShapefileDataStore(this.class.getResource(name))
-        assert shapefile
-        def contents = shapefile.landcover2000
-        assert contents
-        assert 1234 == contents.getCount( Query.ALL )
+    void queryTest() {
+        def str = "";
+        ds.query("SELECT * FROM elements WHERE id > 1")
+                { while(it.next()) {
+                    str+=it.getString('name')+" "+it.getString('number')+" "
+                }}
+        assert "Maybe a complex Name 7455 S N 9272 " == str
+
+        str = "";
+        ds.query("SELECT * FROM elements WHERE id > ?", [1])
+                { while(it.next()) {
```

Comment:
is query available only for jdbc datastore ?

Reply:
What about the same groovy syntaxe  ?

ds.eachRow('select * from PROJECT', 2, 2) { row ->
     println "${row.name.padRight(10)} ($row.url)"
 }

Reply:
1st comment : yes, because only `JDBCDataStore` has the `DataStore` and `Connection` object.
2nd comment : as said in the PR description, other groovy `Sql` methods (such as `earchRow()` ) will come in other PR (to avoid too big PR).